katsdptelstate/aio/test/test_telescope_state.py

```diff
@@ -517,3 +519,20 @@ class TestTelescopeStateRedis(TestTelescopeState):
     async def make_telescope_state(self) -> TelescopeState:
         client = await fakeredis.aioredis.create_redis_pool()
         return TelescopeState(RedisBackend(client))
+
+
+class TestSharedMemoryBackend(asynctest.TestCase):
```

Comment:
Why would you want a shared memory backend?

Reply:
My main thought was that you might want to have connect some async code and some sync code via the same storage e.g. maybe you have some async code you want to test, but don't want to muck about with asyncio in the test itself.

Reply:
I also realised later on that this would allow `load_from_file` via the synchronous backend.

Reply:
Me too - was chasing the vanished load_from_file task through the serpentine labyrinth of commits wondering why it had suffered a bullet.